### DISCUSSION TITLE
Apple pay in PaymentSheet not working iOS 16 emulator for version 5.0.0

### DISCUSSION THREAD

**Discussion by coodysk8er** [flutter-stripe/flutter_stripe#941] [Q&A]
**Describe the bug**
After choosing Apple pay and clicking "Pay", the PaymentSheet dismisses and displays again.
No exception can be caught. It works well with credit cards.

**To Reproduce**
1. Run onn ios 16 enumerator.
2. Display payment sheet.
3. Choose Apply Pay
4. Click "Pay"

**Expected behavior**
Apply payment confirmation should be displayed.

**Smartphone / tablet**
 - Device: iPhone emulator
 - OS: iOS 16
 - Package version: 5.0.0
 - Flutter version 3.3.2 --stable

**Additional context**

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.3.2, on macOS 12.6 21G115 darwin-arm, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.0)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.2)
[✓] VS Code (version 1.71.1)
[✓] Connected device (4 available)
[✓] HTTP Host Availability

• No issues found!

Related Code
```
await  Stripe.instance.initPaymentSheet(
          paymentSheetParameters: SetupPaymentSheetParameters(
            customFlow: false,
            customerEphemeralKeySecret:
                '...customerEphemeralKeySecret...',
            customerId:  '...customerID...',
            paymentIntentClientSecret: '...clientSecret...',
            merchantDisplayName: 'DisplayName',
            applePay: const PaymentSheetApplePay(
              merchantCountryCode: 'US',
              paymentSummaryItems: [
                ApplePayCartSummaryItem.immediate(
                  label: 'Product1',
                  amount: '20.00',
                  isPending: false,
                ),
              ],
            ),
            googlePay: const PaymentSheetGooglePay(
              merchantCountryCode: 'US',
              currencyCode: 'USD',
              testEnv: true
            ),
            style: ThemeMode.light,
            appearance: stripePaymentBottomSheetAppearnce,
          ),
        );

        await Stripe.instance.presentPaymentSheet();
       //repetitively showing apple pay here
```

**Comment by jonasbark** [comment]
Can you confirm this behavior with the example app and server?

**Comment by coodysk8er** [comment]
> Can you confirm this behavior with the example app and server?

I confirmed it worked well on my iPhone 13 pro iOS 16, but not working on the emulator with the same setup
